compiler/modules/front/const.bal

```diff
@@ -326,14 +326,12 @@ function foldBinaryEqualityExpr(FoldContext cx, t:SemType? expectedType, s:Binar
     return expr;
 }
 
-// Remove after JBUG #17977, #32245 is fixed
 function isEqual(SimpleConst c1, SimpleConst c2) returns boolean {
```

Comment:
remove the function

Reply:
Fixed with https://github.com/ballerina-platform/nballerina/pull/847/commits/8495fef3ea1efdbe2d1dd8c19397164205b8ac1c